### PR TITLE
Improve automation

### PR DESCRIPTION
**Pull request by Machine-Maker**
Idea here is to create an issue when a build caused by an automated commit fails to either apply patches successfully or doesn't compile.

### CODE REVIEW

**Comment by Machine-Maker** [comment]
Release sculptor version and update this PR to that version before merging

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -73,35 +94,47 @@ jobs:
...
-      - uses: gradle/gradle-build-action@v2
-        name: Deploy Mache Artifact
-        with:
-          arguments: ':versions:${{ needs.verify.outputs.project_name }}:publishMachePublicationToPaperMCRepository'
+      - uses: gradle/wrapper-validation-action@v2
+        name: Validate Gradle Wrapper
+      - uses: gradle/actions/setup-gradle@v3
+        name: Setup Gradle
+      - name: Deploy Mache Artifact
+        run: ./gradlew ':versions:${{ needs.verify.outputs.project_name }}:publishMachePublicationToPaperMCRepository'
```
Comment (MiniDigger) [inline .github/workflows/deploy.yml:137]: why did you remove the gradle build action here? and I guess we should add --no-daemon

**Review by MiniDigger — CHANGES_REQUESTED** [review]
see discord, lets move the auto updating logic to sculptor, including running the build and opening the issue (but lets keep deploy here in each branch, the running build twice is a fine)
that way we can easier update the auto updating, not having so much logic in each mache branch

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (Machine-Maker) [reply .github/workflows/deploy.yml:137]: gradle build action is obsolete. It says to use setup-gradle and call the script directly. https://github.com/gradle/gradle-build-action

**Pull request closed without merging**